dev-python/pyaudio/pyaudio-0.2.11.ebuild

```diff
@@ -1,9 +1,9 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
```

Comment:
Well, crap. I didn't realize there was an EAPI bump in it as well. You should mention it in the commit summary. 

So yeah... If you want EAPI-7 bump, **do** revbump and drop KEYWORDS. If you do, then also update the ebuild to use `distutils_enable_docs` function which should allow you to remove any `src_` phase from the ebuild altogether, along with `doc?` USE flags. 

I'm not sure if you can commit while editing EAPI-5 ebuild, but you can test it of course.

Reply:
Reverted that to keep it simple.